pkg/apis/core/pods/helpers.go

```diff
@@ -25,19 +25,24 @@ import (
 )
 
 // ContainerVisitorWithPath is called with each container and the field.Path to that container
```

Comment:
Describe the meaning of the return value (can be copied from `pkg/api/v1/pod/util.go`